fauna/vscode
# FE-313 Extension improvements

### Notes
* Update secret config description
* Update secretKey to be adminSecretKey
* Fix child database access
* Update extension logo

### How to test
* Install the extension [faunadb-0.0.1.vsix.zip](https://github.com/fauna/vscode/files/4200288/faunadb-0.0.1.vsix.zip) Updated at 02/13.
* See if the extension image is good
* See if the text on README is good
* Try to access child of child dbs on the sidebar
